### ISSUE TITLE
Attention is not quite all you need

### ISSUE THREAD

**Issue by AshwathSalimath** [PyDataPune/Talks#2]
Hello All,

I currently work as a Graduate Data Scientist at [CeADAR Ireland](https://www.ceadar.ie/) [Day Job]. Also, I am a Research Intern at [Octavian.AI](https://www.octavian.ai/). My interest lies in Graph AI and Predictive Maintenance [Industry 4.0]. This talk will be related to Octavian's work. 

**Talk Overview**

We ([Octavian.AI](https://www.octavian.ai/)) have developed a novel and robust neural network solution to detecting if an item is present in a list. We've created this as an extension of the [popular attention mechanism](https://arxiv.org/pdf/1706.03762.pdf).

**Background** 

While working on [MacGraph](https://github.com/Octavian-ai/mac-graph), a neural network that answers questions using knowledge graphs, we came across a problem: How to tell if something’s not present in a list. 

In traditional programming, it’s easy to tell if something is not in a list: run a for loop over the items, and if you get to the end without finding it, then it’s not there. However, neural networks are not quite so simple. Neural networks are composed of differentiable functions so that they can be trained using [gradient descent](https://en.wikipedia.org/wiki/Gradient_descent). Equality operators, for loops and if conditions, the standard pieces of traditional programming used to solve this task, do not work that well in neural networks. In this talk, I will go through our solution for this task and discuss how it performs reliably across a range of different datasets and architectures.

Check our solution here: https://t.co/oxAe2ZS2Ok

**About Octavian.AI**

We're an open-source research organisation and research new approaches to machine reasoning and graph-based learning.

**Availability for the talk**

Between 25th December 2018 to 15th January 2019.

**I will hope that this talk will get you all excited about Graph AI**

**Comment by PyDataPune** [comment]
accepted for event dated for Jan 05, 2019

**Comment by mayankskb** [comment]
Hey Ashwath,
Please share link of your presentation.
Thanks

**Comment by AshwathSalimath** [comment]
Hello Mayank,

Here is the link of the presentation: https://speakerdeck.com/ashwathsalimath/attention-is-not-quite-all-you-need

Thank you so much for providing us with an opportunity to present our work at PyData Pune. It was a great experience.
